Sources/StreamVideoSwiftUI/Utils/PictureInPicture/StreamPictureInPictureVideoRenderer.swift

```diff
@@ -230,8 +230,12 @@ final class StreamPictureInPictureVideoRenderer: UIView, RTCVideoRenderer {
         let heightDiffRatio = trackSize.height / contentSize.height
         requiresResize = widthDiffRatio >= resizeRequiredSizeRatioThreshold || heightDiffRatio >= resizeRequiredSizeRatioThreshold
         let requiresFramesSkipping = widthDiffRatio >= sizeRatioThreshold || heightDiffRatio >= sizeRatioThreshold
+
+        /// After the recent changes on using Accelerate for frame processing which lead to a big
```

Comment:
nit: Probably better to rephrase this a bit - "recent changes" is a bit relative - we could just mention that skipping frames is subject to a threshold.